web/src/routes/admin/jobs-status/+page.svelte

```diff
@@ -4,28 +4,31 @@
   import Icon from '$lib/components/elements/icon.svelte';
   import UserPageLayout from '$lib/components/layouts/user-page-layout.svelte';
   import { AppRoute } from '$lib/constants';
+  import { asyncTimeout } from '$lib/utils';
   import { getAllJobsStatus, type AllJobStatusResponseDto } from '@immich/sdk';
   import { mdiCog } from '@mdi/js';
   import { onDestroy, onMount } from 'svelte';
   import type { PageData } from './$types';
 
   export let data: PageData;
 
-  let timer: ReturnType<typeof setInterval>;
-
   let jobs: AllJobStatusResponseDto;
 
   const load = async () => {
     jobs = await getAllJobsStatus();
   };
 
+  let running = true;
+
   onMount(async () => {
-    await load();
-    timer = setInterval(load, 5000);
+    while (running) {
+      await load();
```

Comment:
There doesn't seem to be a point in having this `load` function over `jobs = await getAllJobsStatus();`

Reply:
yeah, good catch. updated